_pages/how-we-work/mentorship.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Mentorship
+slack-channel: mentorship
+---
+
+18F is experimenting with providing support for mentorship!  This is formally a part of 18F Learn but it is supported across multiple chapters and business units.
+
+## Getting started
+
+If you'd like to participate in mentorship, it's as simple as reaching out to folks in [#mentorship](https://gsa-tts.slack.com/messages/mentorship)!  Someone will be in touch to review the general information found below and identify what you'd like to focus on and whether you would like to mentor someone, be mentored, or both.
+
+Please note that regardless of how you'd like to participate in mentorship, it is subject to the availability of other folks also interested in participating and focused on your area/skill of interest.
+
+Lastly, the [#mentorship](https://gsa-tts.slack.com/messages/mentorship) channel is also a great place to go to chat with other folks who are currently participating in mentorship or have mentorship experience.  You can learn more about what's currently happening, ask questions, etc.
+
+## Administrative information
+
+We ask that folks commit to a minimum of **6 weeks** to start with once we're able to find a match.  The reason is that it allows for enough time to give the mentorship pair a chance to gel and develop a cadence and relationship with one another.  It's usually a little slow going in the beginning especially if you haven't collaborated with one another before; this is okay!  We want to make sure folks give things a chance and make a concerted effort, and have enough time to achieve their goals.
+
+However, we also don't want folks feeling like they're being forced to do something that isn't working for them or that they are stuck with someone that isn't a great fit.  Whether or not you choose to continue as a pair after the 6 week time period is over is entirely up to both you and your partner.
+
+Prior to starting, we also recommend that folks considering pairing together or being matched up have an intro meeting with each other to review the following things:
+
+* Details pertaining to the interests that you share
+* Scheduling and meeting length preferences
+* What drew you to the person, either to mentor them or to be a mentee of them
+* Establishing a closer bond with someone you normally wouldn’t have the opportunity to do so with
```

Comment:
@ctro hmm, good point.  It's definitely something we've talked about as a benefit of mentorship (meeting and working with someone new!), but now that you mention it, that's not likely something you're going to actively discuss in an intro meeting.

Let's strike it out, but is there any other spot in the documentation that might warrant this notion?  Or should we just forego it entirely?